src/Raven.Server/Web/Operations/OperationsServerHandler.cs

```diff
@@ -24,6 +25,26 @@ await using (var writer = new AsyncBlittableJsonTextWriter(context, ResponseBody
             }
         }
 
+        [RavenAction("/admin/operations/get-operations-id", "GET", AuthorizationStatus.Operator)]
+        public async Task GetOperationsId()
+        {
+            var operations = ServerStore.Operations.GetActive();
+
+            using (ServerStore.ContextPool.AllocateOperationContext(out JsonOperationContext context))
+            {
+                await using (var writer = new AsyncBlittableJsonTextWriter(context, ResponseBodyStream()))
+                {
+                    foreach (Documents.Operations.Operations.Operation operation in operations)
+                    {
+                        writer.WriteStartObject();
+                        writer.WritePropertyName($"Id");
```

Comment:
nameof?

Reply:
This type of operation was unnecessary.

Reply:
I don't follow

I wonder if it is good to use:
```
 writer.WritePropertyName(nameof(operation.Id));
```

Reply:
Sorry, Marcin, I didn't make myself clear.
I understand what you recommended, but I removed this code section entirely from the PR due to irrelevance.